### PR TITLE
Add Manual parse test for WPT WebIDL folder's valid widl files

### PR DESCRIPTION
**Pull request by lukebjerring**
Note: Fails with the following output:

    The following files failed to parse for this repository:
    WebIDL/valid/idl/identifier-qualified-names.widl
    WebIDL/valid/idl/attributes.widl
    WebIDL/valid/idl/iterator.widl
    WebIDL/valid/idl/exception-inheritance.widl

### CODE REVIEW

**Review by mdittmer — COMMENTED** [review]
[incomplete review]

Patching in #105 yields a failure in a file that contains an `exception` entity :thinking:

**Comment by lukebjerring** [comment]
Yah, the list provided [above](https://github.com/GoogleChromeLabs/webidl-diff/pull/101#issue-171796730) had four failures, which are either bugs in the parser or bugs in the IDL content.

**Comment by mdittmer** [comment]
I see. So the hope is that this improves our tests, even if we don't make them all pass before landing?

If so, let's file an issue to track the failures and merge this.

**Comment by lukebjerring** [comment]
Yeah, that's why I made it a manual test.

I think https://github.com/GoogleChromeLabs/webidl-diff/issues/103 and https://github.com/GoogleChromeLabs/webidl-diff/issues/102 cover the 2 issues.

Interestingly, both are used, as you say, in practice, but are not valid according to spec.

File: main/wptTestCasesConfig.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```
Comment (mdittmer) [inline main/wptTestCasesConfig.js:1]: `main/` is generally reserved for (usually non-test) program entry points. This config isn't very long. I think it should be inlined in `test/node/wptIDLFetchAndParse-manual.js`

**Review by mdittmer — APPROVED** [review]
LGTM % what-code-lives-where.

That is, assuming the manual test either passes or fails on meaningful IDL issues.

File: main/wptTestCasesConfig.js (hunk 1 above)
Comment (lukebjerring) [reply main/wptTestCasesConfig.js:1]: For some reason this stops jasmine detecting foam helper and dies.

**Pull request merged**